p2panda-blobs/Cargo.toml

```diff
@@ -20,10 +20,10 @@ bytes = "1.7.1"
 futures-buffered = "0.2.8"
 futures-lite = "2.3.0"
 futures-util = "0.3.30"
-iroh-base = "0.25.0"
-iroh-blobs = { version = "0.25.0", features = ["downloader", "fs-store"], default-features = false }
+iroh-base = "0.26.0"
```

Comment:
I bumped these because the inclusion of the `netwatch` dependency resulted in a conflict with `derive_more`.